samples/dotnet/kernel-syntax-examples/Example20_HuggingFace.cs

```diff
@@ -30,5 +32,9 @@ public static async Task RunAsync()
         var result = await questionAnswerFunction.InvokeAsync("What is New York?");
 
         Console.WriteLine(result);
+        Console.WriteLine(JsonSerializer.Serialize(
+            result.GetHuggingFaceLastResultData(),
```

Comment:
If I'm a skill developer and I call semantic functions from within my skill against a kernel that was given to me, how do I know which extension method to call?

Reply:
For scenarios where you don't know which underlying ITextCompletion service you used, recommend to use pure Json serialization.

```
JsonSerializer.Serialize(textResult.LastPromptResults);
```